Tests/Functional/MissingAllTestsOverride/main.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/MissingAllTestsOverride
+// RUN: %{built_tests_dir}/MissingAllTestsOverride > %t || true
+// RUN: %{xctest_checker} %t %s
+// CHECK: Test Case 'MissingAllTestsOverride.missingAllTestsOverride' started.
+// CHECK: <EXPR>:0: unexpected error: MissingAllTestsOverride.missingAllTestsOverride : threw error "This test case class must override the allTests property" - 
```

Comment:
Awesome!

I don't know if this is technically possible, but is there any way we can get `<EXPR>:0` to instead point to a relevant file and line? It would be *amazing* if we could get this to point to the file and line `class MissingAllTestsOverride` is defined, for example.